### ISSUE TITLE
False Positive `PureRestControllers`

### ISSUE THREAD

**Issue by loganhenson** [tightenco/tlint#32]
```
You should not mix restful and non-restful methods in a controller
```

```php
<?php

namespace App\Http\Controllers;

use App\Data\Models\Profession;

class ProfessionController extends Controller
{
    public function show($slug)
    {
        return view('professions.show')
            ->with('profession', Profession::where('slug', $slug)->firstOrFail())
            ->with('professions', Profession::all());
    }
}

```
